Buscorepuestos/buscorepuestos-web
# Feature/br 352 footer

## What type of PR is this? (check all applicable)


## Description
The footer is redesigned and made responsive, ensuring pixel-perfect compliance, testing, and 100% coverage.

## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-352/footer

## QA Instructions, Screenshots, Recordings

Run the app with pnpm run dev and just see port:3000, verify that it is responsive

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?

none
